gridstatus/ieso.py

```diff
@@ -1076,7 +1080,8 @@ def _get_all_resource_adequacy_jsons(
             )
 
         if last_modified:
-            last_modified = pd.Timestamp(last_modified, tz=self.default_timezone)
+            if last_modified.tz is None:
+                last_modified = last_modified.tz_localize("UTC")
```

Comment:
Do we want to do the same handling here as above?

Reply:
oops, yes.

Reply:
updated.